common/src/test/java/tachyon/resource/ResourcePoolTest.java

```diff
@@ -68,12 +67,14 @@ public void resourcePoolNormalTest() {
   @Test
   public void resourcePoolBlockingTest() throws InterruptedException {
     mThrown.expect(RuntimeException.class);
-    LinkedBlockingQueue queue = PowerMockito.mock(LinkedBlockingQueue.class);
-    TestResourcePool testPool = new TestResourcePool(2, queue);
+    final int POOL_SIZE = 2;
+    @SuppressWarnings("unchecked")
+    LinkedBlockingQueue<Integer> queue = PowerMockito.mock(LinkedBlockingQueue.class);
```

Comment:
Is it necessary to use `PowerMockito.mock` here rather than simple `Mockito.mock`?
It seems `PowerMockito` requires more resources during testing, and if possible, we should use `Mockito` over `PowerMockito`